src/test/java/seedu/address/testutil/TypicalProjects.java

```diff
@@ -56,16 +55,19 @@ public class TypicalProjects {
 
     private TypicalProjects() {} // prevents instantiation
 
+
     /**
      * Returns an {@code AddressBook} with all the typical projects.
      */
-    public static AddressBook getTypicalAddressBook() {
+    /*
+    public static AddressBook getTypicalAddressBookWithProjectOnly() {
         AddressBook ab = new AddressBook();
         for (Project project : getTypicalProjects()) {
             ab.addProject(project);
         }
         return ab;
     }
+    */
```

Comment:
Can keep here for now but at the end of next week we should do a thorough code clean up.